<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QMainWindow" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>180</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Packets received :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>220</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bytes received :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>260</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Last packet size :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>60</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server port :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server IP :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="serverIP">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="serverPort">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>100</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server protocol :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="serverProtocol">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>140</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connected clients :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="numClient">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="packetsReceived">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="bytesReceived">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="lastPacketSize">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="totalPacketsReceived">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total packets received :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="totalBytesReceived">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total bytes received :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="avgPacketSize">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="avglabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Average packet size :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="startServerBtn"/>
   <addaction name="stopServerBtn"/>
  </widget>
  <action name="actionTCP">
   <property name="text">
    <string>TCP</string>
   </property>
  </action>
  <action name="actionUDP">
   <property name="text">
    <string>UDP</string>
   </property>
  </action>
  <action name="startServerBtn">
   <property name="text">
    <string>Start Server</string>
   </property>
   <property name="toolTip">
    <string>Start Server</string>
   </property>
  </action>
  <action name="stopServerBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop Server</string>
   </property>
   <property name="toolTip">
    <string>Stop Server</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
